src/wallet/db.h

```diff
@@ -119,10 +119,14 @@ class CWalletDBWrapper
      */
     void Flush(bool shutdown);
 
+    void IncrementUpdateCounter();
+    unsigned int GetUpdateCounter();
+
 private:
     /** BerkeleyDB specific */
     CDBEnv *env;
     std::string strFile;
+    std::atomic<unsigned int> nUpdateCounter;
```

Comment:
In commit "CWalletDB: Store the update counter per wallet":

Should initialize this value.